.github/workflows/rosa-cluster-auto-provision-on-schedule.yml

```diff
@@ -9,111 +9,48 @@ on:
     uses: ./.github/workflows/rosa-multi-az-cluster-create.yml
     with:
       clusterPrefix: gh-keycloak # ${{ env.CLUSTER_PREFIX }} -- unfortunately 'env.' doesn't work here
+      activeActive: true
       enableExternalInfinispanFeature: true
-      createCluster: false
     secrets: inherit
 
-  run-ap-functional-tests-with-external-infinispan:
-    needs: keycloak-deploy-with-external-infinispan
-    name: Runs the functional test suite in Active/Passive with the External Infinispan feature
+  run-functional-tests-active-active-with-external-infinispan:
+    needs: keycloak-deploy-active-active-with-external-infinispan
+    name: Runs the functional test suite in Active/Active with the External Infinispan feature
     uses: ./.github/workflows/rosa-run-crossdc-func-tests.yml
     with:
+      activeActive: true
       clusterPrefix: gh-keycloak # ${{ env.CLUSTER_PREFIX }} -- unfortunately 'env.' doesn't work here
       skipEmbeddedCaches: true
     secrets: inherit
 
-  run-scaling-benchmark-with-external-infinispan:
-    needs: run-ap-functional-tests-with-external-infinispan
-    name: Scaling Benchmark with Active/Passive External Infinispan Feature
+  run-scaling-benchmark-active-active-with-external-infinispan:
+    needs: run-functional-tests-active-active-with-external-infinispan
+    name: Scaling Benchmark with Active/Active External Infinispan Feature
     uses: ./.github/workflows/rosa-scaling-benchmark.yml
     with:
-      clusterName: gh-keycloak-a # ${{ env.CLUSTER_PREFIX }}-a -- unfortunately 'env.' doesn't work here ${{ env.CLUSTER_PREFIX }}-a
-      skipCreateDataset: true
-      outputArchiveSuffix: 'external-infinispan'
+      clusterName: gh-keycloak-a # ${{ env.CLUSTER_PREFIX }}-a -- unfortunately 'env.' doesn't work here
+      outputArchiveSuffix: 'active-active-external-infinispan'
     secrets: inherit
 
-  keycloak-undeploy-with-external-infinispan:
-    needs: run-scaling-benchmark-with-external-infinispan
+  keycloak-undeploy-active-active-with-external-infinispan:
+    needs: run-scaling-benchmark-active-active-with-external-infinispan
     name: Undeploy Keycloak deployment on the multi-az cluster
     if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
     uses: ./.github/workflows/rosa-multi-az-cluster-undeploy.yml
     with:
       clusterPrefix: gh-keycloak # ${{ env.CLUSTER_PREFIX }} -- unfortunately 'env.' doesn't work here
       skipAuroraDeletion: true
+      activeActive: true
     secrets: inherit
 
   keycloak-deploy-active-active:
-    needs: keycloak-undeploy-with-external-infinispan
+    needs: keycloak-undeploy-active-active-with-external-infinispan
     name: ROSA Scheduled Create Active/Active cluster with Persistent Sessions
     if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
     uses: ./.github/workflows/rosa-multi-az-cluster-create.yml
```

Comment:
isn't `enableExternalInfinispanFeature: true` missing from this config?

Reply:
Hmm. I think you are correct. I thought it is not necessary and `MULTI_SITE` + `PERISTENT_SESSIONS` should be enough to enable the Remote implementation factories for the entities other than sessions. However, it seems not to be true at the moment so we should add it here as well.